syntax = "proto3";

package at.tracklab;

import "types.proto";

service CircuitInfo {
  rpc GetCircuit(CircuitRequest) returns (Circuit) {}
  rpc GetCircuitInformation(CircuitInformationRequest) returns (CircuitInformation) {}
}

message CircuitRequest {
  int32 year = 1;
  int32 round = 2;
}

message CircuitInformationRequest {
  int32 year = 1;
  int32 round = 2;
  string session = 3;
}
